src/components/_LuxLibraryContactInfo.vue

```diff
@@ -74,7 +73,6 @@ export default {
     background-color: var(--color-grayscale-darker);
     a {
       color: var(--color-white);
-      &:hover,
```

Comment:
On hover in dark theme we would want
&:hover {
text-decoration-color: var(--color-princeton-orange-on-black);
}